src/Transposit.ts

```diff
@@ -227,3 +237,26 @@ export interface SignInSuccess {
 export interface OperationParameters {
   [paramName: string]: string;
 }
+
+class EndRequestLogResponse<T> implements OperationResponse<T> {
```

Comment:
Re: value returning undefined: I chose not to do this because then value needs to have type T | undefined, which seems potentially annoying to someone who wrote an operation that always returns a defined value.

Reply:
Everything else about it using the java approach to an interface stemmed from that; needing to have a value getter function requires a defined class (I think?). At that point, I thought it made more sense to do the stuff with the constructor.

Reply:
That's fair enough. I'm just not used to seeing classes used this way in JavaScript.